static/js/components/EducationDisplay.js

```diff
@@ -21,19 +22,19 @@ import {
 import { userPrivilegeCheck } from '../util/util';
 
 export default class EducationDisplay extends ProfileFormFields {
-  openEditEducationForm = index => {
+  openEditEducationForm: Function = index => {
     openEditEducationForm.call(this, index);
-  }
+  };
 
-  openNewEducationForm = (level, index) => {
+  openNewEducationForm: Function = (level, index) => {
     openNewEducationForm.call(this, level, index);
   };
 
-  deleteEducationEntry = () => {
+  deleteEducationEntry: Function = () => {
```

Comment:
idk, which of these do you think is preferable?

first, explicitly annotated the function type signature on the class property:

```js
deleteEducationEntry: (() => void) = () => {
  ...
};
```

or putting the return value annotation on the function literal and just using the `Function` annotation on the class property:

```js
deleteEducationEntry: Function = (): void => {
  ...
};
```

both are valid flow syntax. I'm a little inclined towards the latter because it's maybe a little more consistent, not sure.